src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateCreationSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace WcfTestBridgeCommon
+{
+    [Serializable]
+    public class CertificateCreationSettings
+    {
+        public CertificateCreationSettings()
+        {
+            Subjects = new string[] { string.Empty };
+            IsValidCert = true;
+        }
```

Comment:
Can we use the [C# 6 initialization syntax](https://msdn.microsoft.com/en-us/library/bb384054.aspx?f=255&MSPPError=-2147217396), like

```csharp
    public bool IsValidCert { get; set; } = true; 
```

Or is this syntax not supported in CoreCLR/.NET Native yet?